packs/validator-0.0.40/README.md

```diff
@@ -0,0 +1,80 @@
+# Validator <!-- It must be the pack’s name -->
+
```

Comment:
Can we add a description here on what this pack does? Because, this pack will get synchronized to the registry & will show up on Palette UI. Profile modeler has this new option to show Readme for all the packs and would be good to see a brief description about what this pack is for & a few example configs will be of much help.